exe/3scale-update

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env ruby
+
+require '3scale_toolbox/cli'
+require 'optparse'
+
+options = {}
+
+parser = OptionParser.new do |parser|
+  parser.banner = '3scale update <command> [options]'
+
+  parser.on('-s', '--source SOURCE', "Source") do |domain|
+    options[:source] = domain
+  end
+
+  parser.on('-d', '--destination DESTINATION', "Destination") do |domain|
+    options[:destination] = domain
+  end
+
+  parser.on('-t', '--target-system-name TARGET_SYSTEM_NAME', 'Target system name') do |name|
+    options[:target_system_name] = name
+  end
+
+  parser.on('-h', '--help', 'Prints this help') do
+    puts parser
+    puts
+    puts 'Available Commands:', ['service <service_id>', 'help']
+    exit
+  end
+end
+
+print_help = ->(error = nil) do
+  if error
+    puts "Error: #{error}"
+    puts
+  end
+  parser.parse(['--help'])
+end
+
+
+parser.parse!
+
+def compare_hashes(first, second, keys)
+  keys.map{ |key| first.fetch(key) } == keys.map{ |key| second.fetch(key) }
+end
+
+def fetch_option(options, key)
+  options.fetch(key) { raise OptionParser::MissingArgument, key }
+end
+
+def provider_key_from_url(url)
+  url[/\w*@/][0..-2]
```

Comment:
Any reason why not to use https://github.com/3scale/3scale_toolbox/blob/master/exe/3scale-import#L44-L53 ?